pkg/io/confio/acl.go

```diff
@@ -110,33 +110,30 @@ func aclRules(acl *ir.ACL) []vpcv1.NetworkACLRuleItemIntf {
 	return ruleItems
 }
 
-func updateACL(model *configModel.ResourcesContainerModel, collection *ir.ACLCollection) error {
-	for _, subnet := range model.SubnetList {
-		vpc := subnet.VPC
-		aclName := ScopingString(*vpc.Name, *subnet.Name)
-		acl := collection.ACLs[*vpc.Name][aclName]
-		subnetRef := &vpcv1.SubnetReference{
-			Name:         subnet.Name,
-			CRN:          subnet.CRN,
-			Href:         subnet.Href,
-			ID:           subnet.ID,
-			ResourceType: subnet.ResourceType,
+func updateACL(model *configModel.ResourcesContainerModel, collection *ir.ACLCollection) {
+	var ok bool
+	var acl *ir.ACL
+	var aclItem *configModel.NetworkACL
+
+	for i, subnet := range model.SubnetList {
+		vpcName := subnet.VPC.Name
+		aclName := ScopingString(*vpcName, *subnet.Name)
+
```

Comment:
```suggestion
		vpcName := *subnet.VPC.Name
		aclName := ScopingString(vpcName, *subnet.Name)

```
Avoid dereferencing strings again and again (strings are just pointers anyway)